### ISSUE TITLE
Remote build fails when submitted from Windows

### ISSUE THREAD

**Issue by RolefH** [IBMStreams/streamsx.topology#2381]
Submitting an application with `ContextTypes.STREAMING_ANALYTICS_SERVICE` or `ContextTypes.BUNDLE` with remote build fails:

```
Exception in thread "main" java.lang.IllegalStateException: Error submitting archive for compilation:
"CDISP0095E ERROR: The following path was input, but is not the path to a directory: tk6258766762312045303"
"."
"make: *** [all] Error 1"
"CDISA5488E The 56347f51-ccf9-43fc-b265-9dbbfe5c9d9e build failed for the following domain: production-shared-2."

at com.ibm.streamsx.rest.AbstractStreamingAnalyticsService.buildAndSubmitJob(AbstractStreamingAnalyticsService.java:261)
at com.ibm.streamsx.topology.internal.context.remote.RemoteBuildAndSubmitRemoteContext.submitBuildArchive(RemoteBuildAndSubmitRemoteContext.java:76)
at com.ibm.streamsx.topology.internal.context.remote.RemoteBuildAndSubmitRemoteContext.submitBuildArchive(RemoteBuildAndSubmitRemoteContext.java:30)
at com.ibm.streamsx.topology.internal.context.remote.BuildRemoteContext._submit(BuildRemoteContext.java:63)
at com.ibm.streamsx.topology.internal.context.remote.RemoteContextImpl.submit(RemoteContextImpl.java:30)
at com.ibm.streamsx.topology.context.remote.RemoteContextSubmit.main(RemoteContextSubmit.java:38)
```
This behavior is observed only when the topology is submitted from Python under MS Windows.

**Comment by RolefH** [comment]
Windows: generated **manifest_tk.txt** has CRLF at end of lines. This causes the above error in the following shell command in the makefile:
```
        @set -e; for toolkit in `/bin/cat manifest_tk.txt`; do \
                if [ -d $${toolkit}/opt/python/streams ]; then \
                        PYTHONUSERBASE=${OUT_PY} python3 com.ibm.streamsx.topology/bin/spl-python-extract.py -i $${toolkit};\
                fi; \
                ${STREAMS_INSTALL}/bin/spl-make-toolkit -i $${toolkit}; \
        done;
```
The file must be generated with LF as line separator.

**Comment by RolefH** [comment]
To create the artifacts, `PrintWriter.println(...);` is used, which terminates the line with the *line separator* string, which is defined by the system property **line.separator**, and which is not necessarily a single newline character ('\n'). It must be a Linux line separator as the files are processed by a Linux system.

**Comment by natashadsilva** [comment]
I am having the same problem too.

**Comment by RolefH** [comment]
resolved with [1.14.11 release](https://github.com/IBMStreams/streamsx.topology/releases/tag/v1.14.11)
